python/hail/genetics/reference_genome.py

```diff
@@ -8,31 +8,31 @@
 class ReferenceGenome(object):
     """An object that represents a `reference genome <https://en.wikipedia.org/wiki/Reference_genome>`__.
 
-    :param str name: Name of reference. Must be unique and NOT one of Hail's
-        predefined references: "GRCh37", "GRCh38", and "default".
-
-    :param contigs: Contig names.
-    :type contigs: list of str
-
-    :param lengths: Dict of contig names to contig lengths.
-    :type lengths: dict of str to int
-
-    :param x_contigs: Contigs to be treated as X chromosomes.
-    :type x_contigs: str or list of str
-
-    :param y_contigs: Contigs to be treated as Y chromosomes.
-    :type y_contigs: str or list of str
-
-    :param mt_contigs: Contigs to be treated as mitochondrial DNA.
-    :type mt_contigs: str or list of str
-
-    :param par: List of tuples with (contig, start, end)
-    :type par: list of tuple of (str or int, int, int)
+    Examples
+    --------
 
     >>> contigs = ["1", "X", "Y", "MT"]
     >>> lengths = {"1": 249250621, "X": 155270560, "Y": 59373566, "MT": 16569}
     >>> par = [("X", 60001, 2699521)]
     >>> my_ref = hl.ReferenceGenome("my_ref", contigs, lengths, "X", "Y", "MT", par)
+
+    Parameters
+    ----------
+    name : :obj:`str`
+        Name of reference. Must be unique and NOT one of Hail's
+        predefined references: ``'GRCh37'``, ``'GRCh38'``, and ``'default'``.
+    contigs : :obj:`list` of :obj:`str`
+        Contig names.
+    lengths : :obj:`dict` of :obj:`str` to :obj:`int`
+        Dict of contig names to contig lengths.
+    x_contigs : :obj:`str` or :obj:`list` of :obj:`str`
+        Contigs to be treated as X chromosomes.
+    y_contigs : :obj:`str` or :obj:`list` of :obj:`str`
+        Contigs to be treated as Y chromosomes.
+    mt_contigs : :obj:`str` or :obj:`list` of :obj:`str`
+        Contigs to be treated as mitochondrial DNA.
+    par : :obj:`list` of :obj:`tuple` of (str or int, int, int)
```

Comment:
`str or int` seems like it might cause more trouble than it's worth, but change is likely outside scope